db/migrate/20210513103622_add_sessions_table.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AddSessionsTable < ActiveRecord::Migration[6.1]
+  def change
+    create_table :sessions do |t|
+      t.string :session_id, null: false
+      t.jsonb :data
```

Comment:
1GB session limit!

Reply:
We'll need to keep an eye on how this goes. Our prod db is 100GB currently

Reply:
I guess we can limit it to 1MB if we want?

Reply:
I mean if we are anything close to this numbers than we're doing sth wrong. At the moment we put hard limit of 2MB files for CSV upload so 1MB session limit would not work.